streaming/vision/convert/README.md

```diff
@@ -0,0 +1,119 @@
+    ```
+    python cifar10.py --in_root <Above directory> --out_root <local or remote directory path to save output MDS shard files>
+    ```
+
+## 3. [MS-COCO](https://cocodataset.org/#home)
+
+**Instructions:**
+
+1. Download the COCO 2017 dataset from [here](https://cocodataset.org/#download). Please download both the COCO images and annotations and unzip the files as shown below.
+    <!--pytest.mark.skip-->
+    ```bash
+    mkdir coco
+    wget -c http://images.cocodataset.org/annotations/annotations_trainval2017.zip
+    wget -c http://images.cocodataset.org/zips/train2017.zip
+    wget -c http://images.cocodataset.org/zips/val2017.zip
+
+    unzip annotations_trainval2017.zip
+    unzip train2017.zip
+    unzip val2017.zip
+
+    rm annotations_trainval2017.zip
+    rm train2017.zip
+    rm val2017.zip
+    ```
+
+    That will result in this directory structure:
+    <!--pytest.mark.skip-->
+    ```bash
+    ├── annotations
+    │   ├── instances_train2017.json
+    │   └── instances_val2017.json
+    ├── train2017
+    │   ├── 000000391895.jpg
+    |   |── ...
+    └── val2017
+    │   ├── 000000000139.jpg
+    |   |── ...
+    ```
+
+2. Run the [coco.py](coco.py) script as shown below with the minimalist arguments. The script converts the `train` and `val` dataset split into their own directory. For advanced users, you can look at the supported arguments for [coco.py](coco.py) and change according to your own needs.
+    <!--pytest.mark.skip-->
+    ```
+    python coco.py --in_root <Above directory> --out_root <local or remote directory path to save output MDS shard files>
+    ```
+
+## 4. [ImageNet](https://www.image-net.org/)
+
+**Instructions:**
+
+1. Download the ImageNet dataset from [here](https://image-net.org/download.php). More specifically, 2 files would be needed, `ILSVRC2012_img_train.tar` for training and `ILSVRC2012_img_val.tar` for validation. Then, untar both the files as shown below.
+    <!--pytest.mark.skip-->
+    ```bash
+    mkdir val
+    mv ILSVRC2012_img_val.tar val/
+    tar -xvf ILSVRC2012_img_val.tar -C val/
+    rm ILSVRC2012_img_val.tar
+
+    mkdir train
+    mv ILSVRC2012_img_train.tar train/
+    tar -xvf ILSVRC2012_img_train.tar -C train/
+    rm ILSVRC2012_img_train.tar
+    ```
+
+    That will result in this directory structure:
+    <!--pytest.mark.skip-->
+    ```bash
+    ├── train/
+      ├── n01440764
+      │   ├── n01440764_10026.JPEG
+      │   ├── n01440764_10027.JPEG
+      │   ├── ......
+      ├── ......
+    ├── val/
+      ├── n01440764
+      │   ├── ILSVRC2012_val_00000293.JPEG
+      │   ├── ILSVRC2012_val_00002138.JPEG
+      │   ├── ......
+      ├── ......
+    ```
+
+2. Run the [imagenet.py](imagenet.py) script as shown below with the minimalist arguments. The script converts the `train` and `val` dataset split into their own directory. For advanced users, you can look at the supported arguments for [imagenet.py](imagenet.py) and change according to your own needs.
```

Comment:
```suggestion
2. Run the [imagenet.py](imagenet.py) script as shown below with the minimalist arguments. The script converts the `train` and `val` dataset splits into their own directories. For advanced uses cases, please see the supported arguments for [imagenet.py](imagenet.py) and modify as needed.
```

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/183/commits/4d242aaf5f26f73278a555a3021da6e328d0a28b.